requirements.txt

```diff
@@ -1,35 +1,33 @@
-aiohttp==3.9.5
+aiohappyeyeballs==2.4.4
+aiohttp==3.10.11
 aiosignal==1.3.1
-annotated-types==0.7.0
-async-timeout==4.0.3
-attrs==23.2.0
+async-timeout==5.0.1
+attrs==24.2.0
 autocommand==2.2.2
 backports.tarfile==1.2.0
-certifi==2024.6.2
-chardet==5.2.0
-charset-normalizer==3.3.2
+certifi==2024.8.30
 click==8.1.7
 discord.py==2.4.0
-frozenlist==1.4.1
-idna==3.7
-inflect==7.3.0
-irc==20.4.1
-jaraco.classes==3.4.0
-jaraco.collections==5.0.1
-jaraco.context==5.3.0
-jaraco.functools==4.0.1
+frozenlist==1.5.0
+idna==3.10
+importlib_resources==6.4.5
```

Comment:
Why is this dependency introduced? When I run an update, I don't see this one.

Reply:
Okay, this was introduced because we run Python 3.10, and some deps use this in case you use Python 3.10 (and not newer). Seems part of this lib are now part of stdlib, but not all of them. So this polyfills for older versions.